guides/17_Migration_to_3_0.md

```diff
@@ -159,13 +159,102 @@ handle them please remove.
 - StateMachineTransitionFailed
 - AudioDeviceSwitched
 
+## `MeetingSessionPOSTLogger`
+
+We have renamed `MeetingSessionPOSTLogger` to `POSTLogger` and removed the `MeetingSessionConfiguration` dependency. You don't need to pass the `MeetingSessionConfiguration` object to the `POSTLogger` constructor anymore.
+
+You can pass below `POSTLoggerOptions` to the `POSTLogger` constructor:
+```typescript
+interface POSTLoggerOptions {
```

Comment:
In README.md and this migration guide, we provide JavaScript snippets. JS users won't know `Record` or `interface`.

Reply:
Ok I got it, thought we can use typescript as well. Thanks for the pointer!